site/src/_layoutsPlayground/component.hbs

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>{{default title 'd3fc'}}</title>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/d3fc/{{ package.version }}/d3fc.min.css" rel="stylesheet" />
+</head>
+
+<body>
+    <svg id="chart" width="100%" height="300px"></svg>
+
+    <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/{{ slice package.dependencies.d3 1 }}/d3.js"></script> -->
```

Comment:
The playground uses this to switch to local d3fc.js files when ?local is added
Otherwise gets trimmed out at load